geopy/geocoders/what3words.py

```diff
@@ -139,57 +138,27 @@ def _parse_json(self, resources, exactly_one=True):
         Parse type, words, latitude, and longitude and language from a
         JSON response.
         """
-        if resources.get('error') == "X1":
-            raise exc.GeocoderAuthenticationFailure()
 
-        if resources.get('error') == "11":
+        if 'code' in resources['status']:
             raise exc.GeocoderQueryError(
```

Comment:
Could you please raise `GeocoderAuthenticationFailure` here when `resources['status']['code'] == 401`?

And the same for `reverse` method below.

https://docs.what3words.com/api/v2/#errors